### ISSUE TITLE
Build errors with latest hydrotools

### ISSUE THREAD

**Issue by jameshalgren** [NOAA-OWP/hydrotools#208]
`pip install hydrotools` crashes with the following error. I feel like it might be something in an upstream library and may be identical to the issue [here](https://github.com/NOAA-OWP/hydrotools/issues/198). Documenting for posterity, and just in case there was a version change that broke something. 

```
Requirement already satisfied: h5py in /Users/jshalgren/pyenv3_jupyter/lib/python3.9/site-packages (from h5netcdf>=0.14.0->hydrotools.nwm-client[gcp]>=5.0.1->hydrotools) (3.7.0)
Collecting tables
  Using cached tables-3.7.0.tar.gz (8.2 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [12 lines of output]
      /var/folders/6q/k8j4psmn1nb2w5q19_scthth0000gq/T/H5closeul7iuze_.c:2:5: error: implicit declaration of function 'H5close' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
          H5close();
          ^
      1 error generated.
      cpuinfo failed, assuming no CPU features: No module named 'cpuinfo'
      * Using Python 3.9.6 (default, Aug  5 2022, 15:21:02)
      * Found cython 0.29.32
      * USE_PKGCONFIG: True
      .. ERROR:: Could not find a local HDF5 installation.
         You may need to explicitly state where your local HDF5 headers and
         library can be found by setting the ``HDF5_DIR`` environment
         variable or by using the ``--hdf5`` command-line option.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by jameshalgren** [comment]
@ZacharyWills (we were discussing today...)

**Comment by hellkite500** [comment]
Pretty sure this is a duplicate of the other referenced issue. I just posted a solution that worked for me on that issue, have you tried that?

**Comment by aaraney** [comment]
@jameshalgren, thanks for reporting this! I agree, this seems like an upstream issue. Ill have a look at this tomorrow morning and update this ticket. Hopefully it will be an easy fix.

**Comment by aaraney** [comment]
Can you find and ref that issue @hellkite500?

**Comment by hellkite500** [comment]
#198

**Comment by jameshalgren** [comment]
Yes --- I'm agreeing, this is a duplicate.

**Comment by jameshalgren** [comment]
(And, once I finally figured out where the new OS installation hid my homebrew installation, the proposed fix in #198 worked. )
